cmd/vegawallet/commands/key_isolate.go

```diff
@@ -102,16 +102,22 @@ func BuildCmdIsolateKey(w io.Writer, handler IsolateKeyHandler, rf *RootFlags) *
 		"",
 		"Path to the file containing the wallet's passphrase",
 	)
+	cmd.Flags().StringVarP(&f.IsolatedWalletPassphraseFile,
+		"isolated-passphrase-file", "i",
+		"",
+		"Path to the file containing the new isolated wallet's passphrase",
+	)
```

Comment:
I think we should avoid short options for options that are not common. Otherwise we might end up with different interpretation of `-i` in different commands which is confusing for users. From a user experience point of view, predictability and consistency are very important.

Reply:
I went with `s`? All the good ones are taken!

Reply:
I think we shouldn't have any

Reply:
You can use `.StringVar()` instead of `.StringVarP()`